src/main/scala/MacroUtils.scala

```diff
@@ -0,0 +1,75 @@
+package org.cvogt.flow
+
+import scala.language.experimental.macros
+import scala.reflect.macros.blackbox
+
+import org.cvogt.flow.transforms._
+
+trait FlowMacroBase {
+  val c: blackbox.Context
+
+  val allTransforms = Seq[Transform](
+    ImplicitExtractions,
+    PostfixExtractions,
+    ValidateLazy,
+    Normalize,
+    RewriteExtractions
+  )
+}
+
+trait FlowMacroDebug extends FlowMacroBase { base: FlowMacroBase =>
```

Comment:
I dont think imports will work. The types would not be resolved to be same, since they become path dependent and different views see them as different path based types. I guess we will have to stick with this AFAIK.